src/extension.ts

```diff
@@ -30,7 +32,8 @@ export async function activate(context: ExtensionContext): Promise<AzureAccount>
 	context.subscriptions.push(commands.registerCommand('azure-account.openCloudConsoleWindows', () => cloudConsole(azureLoginHelper.api, 'Windows')));
 	context.subscriptions.push(commands.registerCommand('azure-account.uploadFileCloudConsole', uri => uploadFile(azureLoginHelper.api, uri)));
 	survey(context, reporter);
-	return Promise.resolve(azureLoginHelper.api); // Return promise to work around weird error in WinJS.
+
+	return createApiProvider([azureLoginHelper.api]);
```

Comment:
We need to have a transition period between the versioned api and the old api. One way to accomplish that would be something like `return Object.assign(createApiProvider([azureLoginHelper.api]), azureLoginHelper.api);`. However that's a bit simplistic because ideally any _new_ stuff would only go in the versioned api (somewhat as an incentive for people to move to the versioned api)

Also would be nice to have the following:
- A deprecation warning to extension authors using the old api (maybe in the console? idk)
- Telemetry on how many people are using the old api vs the versioned api